types/tabulator-tables/index.d.ts

```diff
@@ -1606,8 +1609,7 @@ export type Editor =
     | 'range'
     | 'tickCross'
     | 'star'
-    | 'select'
-    | 'autocomplete'
+    | 'list'
```

Comment:
'list' appears twice here - (lines 1612 & 1613) . Is 'select' deprecated?

Reply:
Yes select and autocomplete were replaced with list

Reply:
I see that list is in there twice...not sure how that happened or why TS didn't complain